formatter.go

```diff
@@ -993,6 +1006,10 @@ var defaultTemplateFuncs = template.FuncMap{
 var defaultSuccessTemplate = `[OK] {{ join .LineWidth .AssertPath }}`
 
 var defaultFailureTemplate = `
+{{- range $n, $call := .Stacktrace }}
+{{ $call | indent }}
+{{- end -}}
```

Comment:
Also let's move stacktrace between "test name" and "assertion" and add "trace" header. It will look like this: 

```
        <error message>
        
        test name: TestBlah

        trace:
          <stacktrace here>
        
        assertion:
          Request("GET").Expect().JSON().Object()
        
        actual value:
          <blah>
        
        reference value:
         <blah>
```

This way, error message and test name, which are most important, go first; trace and assertion are grouped together, which makes sense because both of them are kind of traces; assertion and actual/expected/reference are also grouped which is also logical because all them describe assertion made.